src/plone/api/tests/test_content.py

```diff
@@ -209,8 +208,8 @@ def test_create_dexterity(self):
         """Test create dexterity."""
         container = self.portal
 
-        # This section check for DX compatibilty. The custom DX types defined
-        # in plone.api are for Plone 4 compatiblity.
+        # This section check for DX compatibility. The custom DX types defined
+        # in plone.api are for Plone 4 compatibility.
```

Comment:
I know it's not related to this PR but seeing "Plone 4" here made me see that this test and custom type can be removed. This branch is no longer compatible with Plone 4.

This can be done in another PR of course.

Reply:
@wesleybl Feel free to do so after landing this.